sites/fast-website/src/public/index.html

```diff
@@ -3,11 +3,14 @@
   <head>
     <meta charset="utf-8"/>
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="style.css">
     <title><%= htmlWebpackPlugin.options.title %></title>
   </head>
   <body>
-    <fast-design-system-provider use-defaults>
+    <fast-design-system-provider
+      use-defaults
+      background-color="#242424"
+      accent-base-color="#F33378"
```

Comment:
You'll probably also want to change the accentPalette property here also - that's a bit tricky because none of this is in JavaScript. You'll likely to need find this element imperatively and set the palette property or set this up as a custom element that sets that property. You can reference`fast-dna/packages/react/fast-components-styles-msft/generate-palettes.js` for how to do that.

This might be fine for now but just FYI.